riak.cabal

```diff
@@ -121,6 +121,7 @@ library
                 monad-control                 >= 1.0.0.4  && < 1.1,
                 network                       >= 2.3,
                 resource-pool                 == 0.2.*,
+                pipes,
```

Comment:
Would you mind putting a reasonable version range on this? 

`>= 4.0 && < 4.4` perhaps

Reply:
Done, and fixed a bug while I was at it :)